package.json

```
@@ -21,13 +21,14 @@
   "homepage": "https://github.com/tidepool-org/nightwatchtests#readme",
   "dependencies": {
     "dayjs": "1.9.1",
+    "dotenv": "8.2.0",
     "eslint-config-airbnb": "18.2.0",
-    "nightwatch": "1.4.2"
+    "nightwatch": "1.4.2",
+    "otplib": "^12.0.1"
```

I'd recommend locking down package versions
```suggestion
    "otplib": "12.0.1"
```

---

oh gosh i missed this!. thanks!